docs/explanations/motd_messages.md

```diff
@@ -62,13 +103,13 @@ upgraded if that service was enabled. Note that we would deliver the same
 information for `esm-infra` if the service was disabled and the series running
 on the machine is in ESM state.
 
-## MOTD through Ubuntu Pro timer jobs
+## MOTD about important subscription conditions
 
-One of the timer jobs Ubuntu Pro uses can insert additional messages into MOTD.
-These messages will be always delivered before or after the content created by
-the Python script delivered by `update-notifier`. These additional messages are
-generated when `pro` detects that certain conditions on the machine have been
-met. They are:
+One of the [timer jobs](https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/explanations/what_are_the_timer_jobs.html)
+Ubuntu Pro uses can insert additional messages into MOTD.
+These messages will be always delivered close to the content created by
```

Comment:
I'm not sure what "close to" means in this context - is there a more specific word that we can use instead?

Reply:
yep, "... delivered next to the content created by `update-notifier`."
Done